app/assess/templates/macros/sub_criteria_navbar.html

```diff
@@ -0,0 +1,17 @@
+{% macro navbar(sub_criteria, themes, current_theme_id, on_summary) %}
+
```

Comment:
Take from https://github.com/communitiesuk/funding-service-design-assessment/pull/62 for the base template